<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGroupsBox</class>
 <widget class="QGroupBox" name="UserGroupsBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserGroups</string>
  </property>
  <property name="title">
   <string>User Groups</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="lstGroups"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="btnCopy">
       <property name="toolTip">
        <string>Create group from existing</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAdd">
       <property name="toolTip">
        <string>Create group</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Edit group</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delete group</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
